src/Kafka/KafkaInstanceDrawer/KafkaConnectionTab.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  TextContent,
+  Text,
+  TextVariants,
+  Skeleton,
+  ClipboardCopy,
+  Button,
+  Label,
+  Popover,
+  ButtonVariant,
+} from "@patternfly/react-core";
+import { HelpIcon } from "@patternfly/react-icons";
+import { FunctionComponent } from "react";
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+
+export type KafkaConnectionTabProps = {
+  isKafkaPending?: boolean;
+  externalServer?: string;
+  tokenEndPointUrl: string;
+  linkToServiceAccount: string;
+  linkToAccessTab: string;
+  showCreateServiceAccountModal: () => void;
+};
+
+export const KafkaConnectionTab: FunctionComponent<KafkaConnectionTabProps> = ({
+  isKafkaPending,
+  externalServer,
+  tokenEndPointUrl,
+  linkToServiceAccount,
+  linkToAccessTab,
+  showCreateServiceAccountModal,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="mas--details__drawer--tab-content">
```

Comment:
Do we have a corresponding css class somewhere?

Reply:
you are right, css was not added. added it now, the same css is used in KafkaInstanceDrawer.tsx file so imported the it in that file also

Reply:
I see. We really should not import the same css more than once tho, because it will duplicate the same css code for every time we imported it. I'd suggest moving that css code to the drawer component's css file (assuming we have one, else let's create it) and keep the classes as they are in the other components.